docs/3.x/tech/tutorials/exporting-lists.adoc

```diff
@@ -0,0 +1,35 @@
+---
+layout: doc
+title: Exporting Lists and Configurations
+description: Learn how to export your lists and Configurations in CleanSpeak
+---
+
+
+== Exporting Lists
+Note: Making changes to your lists after exporting will cause issues when importing.
+
+
+You may be asked by our support team to export your lists and configurations so that we can test it on our end.
+Maybe you're moving lists between systems, or you're doing an upgrade and you want to backup your CleanSpeak DB.
+Here we will show you how to quickly and easily export your lists and configurations in CleanSpeak.
```

Comment:
This doesn't backup the CleanSpeak database. It is only an export of your list configuration.